### PR TITLE
[Unit Tests] Fix the build of xnnpack-based tests

### PR DESCRIPTION
**Pull request by weimingzha0**
Also increase the parallelism of tensorrt tests

### CODE REVIEW

**Comment by weimingzha0** [comment]
@xuhongyao the main() are not generated for xnnpack-based tests.  Do you know why?

**Comment by xuhongyao** [comment]
> @xuhongyao the main() are not generated for xnnpack-based tests. Do you know why?


Is it not generated by other devices? If there is an error in halo compilation, it will not be generated

**Comment by weimingzha0** [comment]
> > @xuhongyao the main() are not generated for xnnpack-based tests. Do you know why?
> 
> Is it not generated by other devices? If there is an error in halo compilation, it will not be generated

Oh. I see. For "interpret mode", GenerateTestFunc is called too late. When emitting code for "interpret mode", the code returns earlier.

File: lib/target/generic_cpp/generic_cxx_codegen.cc [hunk 1]
```diff
@@ -620,22 +622,41 @@ void GenericCXXCodeGen::RunOnFunction(Function& function) {
...
+        os_ << "(";
+        bool is_first = true;
+        for (int i = 0, e = function.Args().size(); i < e; ++i) {
+          os_ << (is_first ? "" : ", ");
+          os_ << "(const float*)inputs[" << i << "]";
+          is_first = false;
+        }
+        for (int i = 0, e = return_inst->GetNumOfOperands(); i < e; ++i) {
+          os_ << (is_first ? "" : ", ");
+          os_ << "(float*)outputs[" << i << "]";
```
[the remark this answers is not in the document]
Comment (weimingzha0) [reply lib/target/generic_cpp/generic_cxx_codegen.cc:655]: right

**Review by xuhongyao — APPROVED** [review]

**Pull request merged**